libraries/rust/src/schema/ticker.rs

```diff
@@ -1,8 +1,66 @@
+use crate::schema::prelude::{Price, Volume};
+use crate::schema::symbol::Ticker;
+use chrono::prelude::NaiveDate;
+use color_eyre::Result;
 use serde::{Deserialize, Serialize};
+use std::env;
+use tracing::debug;
 
 #[derive(Debug, Serialize, Deserialize, PartialEq)]
-pub struct Ticker {
-    symbol: String,
+pub struct Symbol(String);
+
+impl std::fmt::Display for Symbol {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+pub enum TickerStatus {
+    #[serde(rename = "OK")]
+    Ok,
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+pub struct OHLCV {
```

Comment:
This can also be called `Bar` as in "bar data" another term for OHLCV stuff if you prefer that. Just an option.

Reply:
Bar 👍